src/utils/clmfates_interfaceMod.F90

```diff
@@ -562,8 +566,6 @@ subroutine init(this, bounds_proc )
       use clm_instur       , only : wt_nat_patch
       use FATESFireFactoryMod , only: create_fates_fire_data_method
 
-      implicit none
```

Comment:
that's odd, its removal was an oversight, I will put it back.

Reply:
actually, we have a call to implicit none at the top of the module, this should apply it to the whole module I believe, no?  My take was that declaring it in each routine was redundant.

https://github.com/ESCOMP/CTSM/blob/79a77d6eb3e3ca1b16e8abb29a68b844221bc4ee/src/utils/clmfates_interfaceMod.F90#L165

Reply:
You are correct that declaring it at the top of the module -- should be -- enough. But, in the past we did see compilers that didn't handle this correctly and you needed it in every subroutine. It's long enough ago that I assume that's been fixed now, but I can't say for sure. So I can go either way on this one. I kind of figure we shouldn't explicitly add it for each subroutine, but maybe not take it away from code that already have it? So a leave it alone type approach....

Reply:
Since, this is something that should be handled now, we'll leave it like it is:

https://stackoverflow.com/questions/24337413/where-to-put-implicit-none-in-fortran